serenity-core/src/main/java/net/thucydides/core/configuration/WebDriverConfiguration.java

```diff
@@ -58,19 +58,4 @@ private void throwUnsupportedDriverExceptionFor(final String driverType) {
                 + " is not a supported browser. Supported driver values are: "
                 + SupportedWebDriver.listOfSupportedDrivers());
     }
-
-    @Override
-    public WebDriverConfiguration copy() {
```

Comment:
Removed the copy function - it was only used as a workaround to update the environment variables